tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -1297,11 +1312,62 @@ public void TestDescription()
             {
                 BaseColumn column = description.Column(i);
                 Assert.Equal(4, column.Length);
-                Assert.Equal((float)10, column[0]);
+                Assert.Equal((float)9, column[0]);
```

Comment:
Why did this change?

Reply:
Because `Description()` now returns `Length - NullCount` instead of just `Length`. In a notebook, this will help identify which columns have nulls.